### PR TITLE
Fix `ValueError` in `get_column_plot` if column is datetime string

### LINKED ISSUE #309: Method `get_column_plot`: `ValueError` if a datetime column is formatted as a string
### Environment Details

* SDMetrics version: 0.9.1
* Python version: Any
* Operating System: Any

### Error Description

Starting from SDV 1.0, users will provide a new metadata format. The current version of SDMetrics (0.9.1) already supports this format.

However, I am having an issue when the datetime columns are present in a string format such as: `"27 Dec 2020"` (format string =`"%d %b %Y"`)

### Steps to reproduce
Replicate using SDV 1.0

```python
from sdv.datasets.demo import download_demo
from sdv.lite import SingleTablePreset
from sdv.evaluation.single_table import get_column_plot

real_data, metadata = download_demo(
    modality='single_table',
    dataset_name='fake_hotel_guests'
)

synthesizer = SingleTablePreset(metadata, name='FAST_ML')
synthesizer.fit(real_data)
synthetic_data = synthesizer.sample(num_rows=500)

fig = get_column_plot(
    real_data=real_data,
    synthetic_data=synthetic_data,
    column_name='checkin_date',
    metadata=metadata
)

```

Output
```
ValueError: invalid literal for int() with base 10: '27 Dec 2020'
```

### PR DESCRIPTION
**Pull request by frances-h**
Resolve #309

### CODE REVIEW

**Review by pvk-developer — APPROVED** [review]
LGTM! 👍🏻

File: sdmetrics/reports/utils.py [hunk 1]
```diff
@@ -239,9 +262,15 @@ def get_column_plot(real_data, synthetic_data, column_name, metadata):
...
         raise ValueError(f"Column '{column_name}' not found in synthetic table data.")
 
+    column_meta = columns[column_name]
     sdtype = get_type_from_column_meta(columns[column_name])
-    real_column = real_data[column_name]
-    synthetic_column = synthetic_data[column_name]
+    if sdtype == 'datetime':
+        datetime_format = column_meta.get('format') or column_meta.get('datetime_format')
+        real_column = convert_to_datetime(real_data[column_name], datetime_format)
+        synthetic_column = convert_to_datetime(synthetic_data[column_name], datetime_format)
```
Comment (amontanez24) [inline sdmetrics/reports/utils.py:270]: This logic seems similar to what's on line 448. Can we have a `convert_datetime_columns` function that takes in the real data column, synthetic data column and column metadata and converts the column if necessary or is the logic too different?

File: sdmetrics/reports/utils.py [hunk 2]
```diff
@@ -265,9 +285,9 @@ def get_column_plot(real_data, synthetic_data, column_name, metadata):
     column_meta = columns[column_name]
     sdtype = get_type_from_column_meta(columns[column_name])
     if sdtype == 'datetime':
-        datetime_format = column_meta.get('format') or column_meta.get('datetime_format')
-        real_column = convert_to_datetime(real_data[column_name], datetime_format)
-        synthetic_column = convert_to_datetime(synthetic_data[column_name], datetime_format)
+        real_column, synthetic_column = convert_datetime_columns(real_data[column_name],
+                                                                 synthetic_data[column_name],
```
Comment (amontanez24) [inline sdmetrics/reports/utils.py:289]: minor: I think the format you used below is nicer

**Review by amontanez24 — APPROVED** [review]
:shipit:

**Pull request merged**